scipy/signal/tests/test_spectral.py

```diff
@@ -1092,14 +1093,21 @@ def test_normalize(self):
         pgram = lombscargle(t, x, f)
         pgram2 = lombscargle(t, x, f, normalize=True)
 
+        # Calculate the scale to convert from unnormalized to normalized
+        weights = np.ones_like(t)/float(t.shape[0])
+        Y_sum = (weights * x).sum()
+        YY_hat = (weights * x * x).sum()
+        YY = YY_hat - Y_sum * Y_sum
+        scale_to_use = 2/(YY*t.shape[0])
+
         # check if normalization works as expected
-        assert_allclose(pgram * 2 / np.dot(x, x), pgram2)
+        assert_allclose(pgram * scale_to_use, pgram2)
```

Comment:
I had to change the scale just slightly, as numerically (although not algebraically—when there is no offset) there will be a tiny difference.